### PR TITLE
fix: Missing windowId from network events

### PR DESCRIPTION
**Pull request by benjackwhite**
## Problem

I had a bad `as` type conversion which masked a bad property.

## Changes

* Fixes this so now the right window is associated again

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: frontend/src/loadPostHogJS.tsx [hunk 1]
```diff
@@ -27,6 +27,10 @@ export function loadPostHogJS(): void {
                 bootstrap: !!window.POSTHOG_USER_IDENTITY_WITH_FLAGS ? window.POSTHOG_USER_IDENTITY_WITH_FLAGS : {},
                 opt_in_site_apps: true,
                 loaded: (posthog) => {
+                    if (posthog.webPerformance) {
```
Comment (pauldambra) [inline frontend/src/loadPostHogJS.tsx:30]: is this leftover from debugging?

**Review by pauldambra — APPROVED** [review]
:shipit:

File: frontend/src/loadPostHogJS.tsx (hunk 1 above)
Comment (benjackwhite) [reply frontend/src/loadPostHogJS.tsx:30]: No this is a fix that I forgot to add after we made network recording disabled by default on localhost

Comment (benjackwhite) [reply frontend/src/loadPostHogJS.tsx:30]: Would be good to display something in the UI for these cases...

**Pull request merged**